src/__experimental__/components/simple-color-picker/color-option/color-option.spec.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import 'jest-styled-components';
+import TestRenderer from 'react-test-renderer';
+import ColorOption from './color-option.component';
+import StyledColorOption from './style/color-option.style';
+import StyledColorSampleBox from '../color-sample-box/style/color-sample-box.style';
+import { assertStyleMatch } from '../../../../__spec_helper__/test-utils';
+import { rootTagTest } from '../../../../utils/helpers/tags/tags-specs/tags-specs';
+import classicTheme from '../../../../style/themes/classic';
+
+function render(props) {
+  return shallow(
+    <ColorOption
+      name='color-picker'
+      color='#0073C2'
+      { ...props }
+    />
+  );
+}
+
+function renderStyles(props) {
+  return TestRenderer.create(
+    <StyledColorOption
+      name='color-picker'
+      color='#0073C2'
+      { ...props }
+    />
+  );
+}
+
+describe('ColorOption', () => {
+  let wrapper;
+
+  it('contains input and color sample box', () => {
+    wrapper = render();
+    expect(wrapper.children()).toHaveLength(2);
```

Comment:
this test checks if there are two children, which does not correspond to its description